catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -580,6 +584,25 @@ private String getBindingTypeToActivate(List<ServiceBindingType> bindingTypes) {
     return bindingTypeToActivate;
   }
 
+  private Dictionary<String, Object> copyConfigProperties(
+      Dictionary<String, Object> properties, String factoryPid) {
+    Dictionary<String, Object> copiedProperties = new Hashtable<>();
+    ObjectClassDefinition objectClassDefinition = getObjectClassDefinition(factoryPid);
+    if (objectClassDefinition == null) {
+      LOGGER.debug(
+          "ObjectClassDefinition not found for factoryPid: {}. Unable to copy properties.",
+          factoryPid);
+      return copiedProperties;
+    }
+    Stream.of(objectClassDefinition)
```

Comment:
❗️ This will cause the registry id stored in the properties to be lost! The registry id is added to the properties so the registry can track with sources belong to which registry but that property is not part of the object class definition. You could maybe just copy that property separately.